Examples/Detectors/TGeoDetector/include/ACTFW/TGeoDetector/TGeoDetectorOptions.hpp

```diff
@@ -165,33 +165,33 @@ std::vector<Acts::TGeoLayerBuilder::Config> readTGeoLayerBuilderConfigs(
       vm["geo-tgeo-ringtolerance"].template as<read_range>();
 
   // The layer names to parse for in the TGeo
-  read_strings nlayernames =
-      vm["geo-tgeo-nlayernames"].template as<read_strings>();
-  read_strings clayernames =
-      vm["geo-tgeo-clayernames"].template as<read_strings>();
-  read_strings playernames =
-      vm["geo-tgeo-playernames"].template as<read_strings>();
+  read_strings nvolumename =
+      vm["geo-tgeo-nvolume-name"].template as<read_strings>();
+  read_strings cvolumename =
+      vm["geo-tgeo-cvolume-name"].template as<read_strings>();
+  read_strings pvolumename =
+      vm["geo-tgeo-pvolume-name"].template as<read_strings>();
 
-  std::array<read_strings, 3> layernames = {nlayernames, clayernames,
-                                            playernames};
+  std::array<read_strings, 3> volumename = {nvolumename, cvolumename,
```

Comment:
Could you CamelCase the variables on the left and right side?

Reply:
I CamelCased the entire file (it was not consistently done).